Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN CGO_ENABLED=0 GOOS=linux \
 FROM alpine:latest
 RUN addgroup -S appgroup && adduser -S appuser -G appgroup
```

Comment:
Maybe we could replace this with `useradd -m appuser`? This can help us avoid explicitly setting the `$HOME` environment variable. The `-U` flag can be passed to create a group too.

Reply:
Useradd does not exist on alpine. The $HOME variable is always set to the home dir of the current user, so if we were to login as appuser, $HOME would be /hone/appuser. Now since we are running as the host user, the container thinks we are "unknown user with id 1000" for example. As such the $HOME variable will not be set correctly.

Reply:
That makes sense.